### PR TITLE
Update ai_platform.ipynb

### PR DESCRIPTION
**Pull request by gogasca**
Update to ai-platform command instead of legacy ml-engine

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3260)

### CODE REVIEW

File: samples/core/ai_platform/ai_platform.ipynb [hunk 1]
```diff
@@ -345,4 +345,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
\ No newline at end of file
+}
```
Comment (numerology) [inline samples/core/ai_platform/ai_platform.ipynb]: Do you mind removing the new \n in the end?

Comment (numerology) [reply samples/core/ai_platform/ai_platform.ipynb]: I mean adding back.

Comment (gogasca) [reply samples/core/ai_platform/ai_platform.ipynb]: Got it, I edited with GitHub file editor, looks like it does that by default.

**Pull request merged**